lib/vagrant.pm

```diff
@@ -0,0 +1,34 @@
+package vagrant;
+use testapi;
+use strict;
+use warnings;
+use utils;
+
+our @ISA    = qw(Exporter);
+our @EXPORT = qw(setup_vagrant);
+
+# - install vagrant and all additional packages
+# - launch the required daemons
+#
+# Named parameters:
+# - install_virtualbox: bool
```

Comment:
If that bool is true, it does an entirely different thing, so IMO having separate methods makes more sense.

Reply:
I've refactored the function into two separate ones and started a new test run: https://openqa.opensuse.org/tests/1017642